lib/framework.js

```diff
@@ -0,0 +1 @@
+module.exports = require('@livingdocs/framework-sdk-prebuild')
```

Comment:
Why do you use a custom path?

Reply:
To hide the 'framework-sdk-prebuild' part a bit. As it's not intended that someone directly requires the framework-prebuild. But it was just a hunch. Not sure if it actually improves anything...

Reply:
Exposing `livingdocs-node-sdk/lib/framework` is probably not better 😁 
And people will land here first anyways: https://www.npmjs.com/package/@livingdocs/framework-sdk-prebuild

Reply:
...maybe I'll add a README in the prebuild as well stating to use the SDK instead.